### PR TITLE
chore: project addition by Harshu467 #1828

### PR DESCRIPTION
**Pull request by Harshu467**
## Description

I have Added React-Native Project as I describe in issue #1828 

## Screenshots
I am sharing video of that project UI

https://github.com/priyankarpal/ProjectsHut/assets/89975347/2c110592-4fb3-4bc7-ab27-2d95b1f9420e

### CODE REVIEW

**Comment by Harshu467** [comment]
Please Check and Merge @priyankarpal

File: DB/projects.json [hunk 1]
```diff
@@ -3634,7 +3634,23 @@
...
+      },
+      {
+        "link": "https://github.com/Harshu467/NFT",
+        "title": "NFT Market",
+        "description": "Master React Native by building a modern NFT Marketplace iOS and Android #ReactNative Application ",
+        "tech": [
+          "html",
+          "css",
+          "javascript",
+          "reactjs JS",
```
Comment (priyankarpal) [inline DB/projects.json:3646]: remove `JS`
```suggestion
          "reactjs",
```

File: DB/projects.json [hunk 2]
```diff
@@ -3634,7 +3634,23 @@
...
+          "useContext",
```
Comment (priyankarpal) [inline DB/projects.json:3647]: I think `useContext` not needed 
```suggestion
       
```

File: DB/projects.json [hunk 3]
```diff
@@ -3634,7 +3634,23 @@
...
+          "API",
```
Comment (priyankarpal) [inline DB/projects.json:3648]: convert to lowercase 
```suggestion
          "api",
```

File: DB/projects.json [hunk 4]
```diff
@@ -3634,7 +3634,23 @@
...
+          "Vercel",
```
Comment (priyankarpal) [inline DB/projects.json:3649]: just you need to mention tech stacks (e.g. reactjs, nodejs, not vercel, usecontext  api )
```suggestion
    
```

File: DB/projects.json [hunk 5]
```diff
@@ -3634,7 +3634,23 @@
...
+          "React-Native"
```
Comment (priyankarpal) [inline DB/projects.json:3650]: convert to lowercase

```suggestion
          "reactnative"
```

**Review by priyankarpal — CHANGES_REQUESTED** [review]

File: DB/projects.json (hunk 1 above)
Comment (Harshu467) [reply DB/projects.json:3646]: Changes Done

File: DB/projects.json (hunk 2 above)
Comment (Harshu467) [reply DB/projects.json:3647]: Removed

File: DB/projects.json (hunk 3 above)
Comment (Harshu467) [reply DB/projects.json:3648]: Done

**Review by priyankarpal — APPROVED** [review]
Looks Good 🎉

**Pull request merged**